src/execution.js

```diff
@@ -68,7 +68,9 @@ const wrapMeasureIndexesFromMappings = (metricMappings, headers) => {
  *                 property "where" containing query-like filters
  *                 property "orderBy" contains array of sorted properties to order in form
  *                      [{column: 'identifier', direction: 'asc|desc'}]
- * @param {Object} settings - AJAX settings
+ * @param {Object} settings - AJAX settings. Set "extended" to true to retrieve the result
```

Comment:
I wouldn't call this parameter `AJAX settings`, cause it might mislead people to think they may adjust `xhr` with it. Just `settings` will do.